kyo-scheduler/jvm/src/main/scala/kyo/scheduler/Scheduler.scala

```diff
@@ -212,10 +217,53 @@ final class Scheduler(
             receiver.gauge(scope, "flushes")(flushes.sum().toDouble)
         )
     }
+
+    def status(): Scheduler.Status = {
```

Comment:
The scheduler classes now expose `status` methods, which can be useful outside of the `top` utility as well.